modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/service/impl/CommerceShippingFixedOptionLocalServiceImpl.java

```diff
@@ -125,15 +128,29 @@ public CommerceShippingFixedOption deleteCommerceShippingFixedOption(
 			deleteCommerceShippingFixedOptionRels(
 				commerceShippingFixedOption.getCommerceShippingFixedOptionId());
 
+		// Commerce shipping option account entry rels
+
+		_commerceShippingOptionAccountEntryRelLocalService.
+			deleteCommerceShippingOptionAccountEntryRelsByCSFixedOptionKey(
```

Comment:
Ok for now, but we should stop using this approach, with services that calls other services (sometimes even in different modules)
We should instead use model listeners to manage linked entites deletion.

Reply:
We can't do that here, it will create a circular dependency